frontend/bhrigu-lake-trek/index.html

```
@@ -0,0 +1,514 @@
+    <footer class="site-footer">
+        <div>
+            <strong>Incredible India Explorer</strong>
+            <p>Bhrigu Lake Trek · Himachal Pradesh</p>
+        </div>
+        <a href="#main-content">Back to top ↑</a>
+    </footer>
+
+    <dialog id="lightbox" class="lightbox" aria-label="Image viewer">
+        <button class="lightbox-close" type="button" aria-label="Close image viewer">×</button>
+        <div class="lightbox-content">
+            <img id="lightbox-image" src="" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)